src/site/_data/contributors.js

```diff
@@ -64,6 +64,19 @@ const contributors = {
       en: 'V8 JavaScript whisperer',
     },
   },
+  hongchanchoi: {
```

Comment:
@hoch - let me know if there's anything else you want added here.

Reply:
Does a profile photo already exist for @hoch? If not, here are the instructions
https://web.dev/handbook/contributor-profile/#create-a-profile-image

Reply:
Yep - it does! It must've been moved over from WebFu? The correct images are already there.

Reply:
I am okay with using the same photo. :)

Reply:
got it, thanks!